<?xml version="1.0"?>
<launch>
  <!-- Load controller parameters -->
  <rosparam command="load" file="$(find fetch_drivers)/config/whole_body_controller_mpc.yaml" ns="fetch_whole_body_controller" />
  
  <!-- Launch the controller node using Python node -->
  <node name="fetch_whole_body_controller" pkg="fetch_drivers" 
        type="whole_body_controller_mpc" 
        output="screen">
    <!-- Parameters loaded from config file -->
  </node>
</launch> 